src/utils/state.mock.js

```diff
@@ -9,18 +9,30 @@ import reducer from '../reducers';
 import { updateFontLoaded } from '../actions';
 import { getGraphInput } from '../selectors/layout';
 import { updateGraph } from '../actions/graph';
+import { toggleTypeDisabled } from '../actions/node-type';
 import { graphNew, graphDagre } from './graph';
 
 /**
  * Prime the state object for the testing environment
- * by running the asynchronous actions synchronously
+ * by running the asynchronous actions synchronously.
+ * Optionally apply additional actions before or after layout.
  * @param {Object} props
+ * @param {?Function[]} props.beforeLayoutActions Functions that given state return actions to reduce before layout
+ * @param {?Function[]} props.afterLayoutActions Functions that given state return actions to reduce after layout
  */
-export const prepareState = (props) => {
+export const prepareState = ({
+  beforeLayoutActions = [],
+  afterLayoutActions = [],
+  ...props
+}) => {
   const initialState = getInitialState(props);
   const actions = [
     // Set fontLoaded = true:
     () => updateFontLoaded(true),
+    // Enable parameters as they are disabled by default:
+    () => toggleTypeDisabled('parameters', false),
```

Comment:
Please add within the comments that this is a temporary setup and will need to be taken out once the failing tests due to the new parameter setup are revisited :)